.travis.yml

```diff
@@ -5,13 +5,11 @@ rvm:
 addons:
   postgresql: "9.3"
 
-before_script:
-  - git clone https://github.com/mezuro/kalibro_install.git -b v3.5 kalibro_install
-  - pushd kalibro_install
-    # Remove bugged libzmq3 package, see https://github.com/travis-ci/travis-ci/issues/982 and https://github.com/travis-ci/travis-ci/issues/1715 for details
+before_script: 
+  # Remove bugged libzmq3 package, see https://github.com/travis-ci/travis-ci/issues/982 and https://github.com/travis-ci/travis-ci/issues/1715 for details
   - sudo apt-get remove libzmq3
-  - bash install.sh
-  - popd
+  - git clone https://github.com/mezuro/kalibro_install.git -b v3.6 kalibro_install
+  - KALIBRO_PROCESSOR_VERSION=hotspot_metrics_list ./kalibro_install/install.sh
```

Comment:
We need to release another version of kalibro processor and update kalibro_install